### ISSUE TITLE
VAC is banning this cheat

### ISSUE THREAD

**Issue by arthaswow** [RatPoison-dev/RatPoison#794]
I get VAC ban 5 days after using this cheat =)

**Comment by sotakoira** [comment]
what version did you use, account, config etc

**Comment by Worass** [comment]
> I get VAC ban 5 days after using this cheat =)

Iq issue. What settings did you use? Send proof of the whole information table. I doubt it's even a VAC.

**Comment by Aggelos4you** [comment]
> ποια έκδοση χρησιμοποιήσατε, λογαριασμό, διαμόρφωση κ.λπ.

I have VAC banned too and I used the version 1.8.5.7

**Comment by Worass** [comment]
Yeah, but y'all still don't prove the points that you have. Please provide the proof.

**Comment by arthaswow** [comment]
> what version did you use, account, config etc

1.7.1.3 on release

i used all options possible, and skin change too
https://i.imgur.com/JcVwAW5.png

its the only hack i ever used

**Comment by SPRAVEDLIVO** [comment]
cope

**Comment by Blob972** [comment]
Is this real bann pls ? Need to know if i keep use it ? :)

**Comment by sotakoira** [comment]
staff isnt banned yet but i dont advise to cheat

**Comment by mountaincastle47** [comment]
i lost my main lol

**Comment by Blob972** [comment]
> 
> 
> i lost my main lol

What version did u use and what future ? Do u have a link of your steam account pls ? :)

**Comment by Blob972** [comment]
feature*

**Comment by sotakoira** [comment]
Guys just post:

- Last time you used the cheat
- Did your trust factor drop before the ban?
- Version of the cheat (exact) 
- Enabled features
- Did you cheat on alts with other cheats?
- Links to banned accounts

I have only used G37OBS (a year ago) and RatPoison 1.8.4.1, my trust is fine and I'm not banned.

**Comment by jovi-py** [comment]
i also got a VAC ban but i'm not sure if it was this cheat. i used a different cheat on both my alts and got banned on both. but i never used rat poison on one of the alts.

**Comment by Gary4u** [comment]
> Guys just post:
> 
> * Last time you used the cheat
> * Did your trust factor drop before the ban?
> * Version of the cheat (exact)
> * Enabled features
> * Did you cheat on alts with other cheats?
> * Links to banned accounts
> 
> I have only used G37OBS (a year ago) and RatPoison 1.8.4.1, my trust is fine and I'm not banned.

> * 1 day before get ban
> * Trust Factor turned yellow and the next day red, after 2 days i VAC banned 
> * Rat Poison 1.8.5.7
> * Wallhack Radar "Link cfg" https://file.io/ysO1VLgv8Rcs
> * no, i don't use other cheat
> * s.team/p/fqwg-cqtr

**Comment by deivuks420yxz** [comment]
I can confirm also, that one guy was banned using this cheat :D

**Comment by MorphuesI** [comment]
I also lost my main and another acc. On my main I got VAC after a week of that rumor (ratpoision got patched) and then I got VAC on second one too after a week.

I used Aim, Chams and Skeleton

On my main acc my trust got low and after a week or two I got VAC, even tho I wasn't playing cs. 
But on the 2nd acc trust factor was fine and I got VAC after a week

I used version 1.7.1.3

https://steamcommunity.com/profiles/76561198259135233 (main)
https://steamcommunity.com/profiles/76561198061937507 (second one)
